docs/command_context.rst

```diff
@@ -72,3 +68,10 @@ Context variables only available to Jobs
     The last successful run date (defaults to current date if there was no
     previous successful run). Supports date arithmetic using the form
     ``{last_success#shortdate-1}``.
+
+**manual**
+    ``true`` if the job was run manually. ``false`` otherwise.
```

Comment:
I was a little confused by this explanation. I suggest giving an example of "manual" usage, in this case `tronctl start`, and an example of how it could be used to enable feature like verbose logging.